corehq/apps/accounting/migrations/0077_add_web_user_feature.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations
+from django.core.management import call_command
+
+from corehq.privileges import LOADTEST_USERS
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.apps.accounting.bootstrap.utils import _ensure_feature_rates
+from corehq.apps.accounting.bootstrap.config.web_user_feature_rate import BOOTSTRAP_CONFIG
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
```

Comment:
I am not clear about the functionality here, can you help me in understanding why is this function required?

Reply:
Hi Amit, thank you for asking this! I wasn't clear why this is required when I wrote it. I saw it in this [migration](https://github.com/dimagi/commcare-hq/blob/20f4ae094dfbeff463c1d837a30b095b8cccc076/corehq/apps/accounting/migrations/0046_new_plans.py#L15C23-L23) so I thought this is required for all migrations touching the software plans. But after learning more about privilege, I think I don't need this. Remove it here: 4afa80ca09a622b7d7e22ddc6260b8d41b9bb82c